docs/how-to/cluster-cfg.rst

```diff
@@ -0,0 +1,60 @@
+Configure cluster_network/ public_network
+=========================================
+
+
+Overview
+--------
+Network configuration is critical for building a high performance Ceph Storage Cluster. Since, Ceph Clients make requests directly to Ceph OSD Daemons. Ceph OSD Daemons perform data replication on behalf of Ceph Clients, which means replication and other factors impose additional loads on Ceph Storage Cluster networks. Refer to `Ceph Network Config <https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/>`_.
+
+Implementation
+--------------
+Microceph cluster config subcommands rely on ‘ceph config’ as the single source of truth for config values and use the same for getting/setting the configs. After updating (setting/resetting) a config value the same request is forwarded to other hosts on the microceph cluster for restarting particular daemons on the host so that updated configs can take effect.
```

Comment:
For a How-To I'd suggest leaving out details of the implementation. Maybe only mention that affected services will be restarted -- this could be moved to the end of the `Usage` section though.

Reply:
yeah, exactly my thought. I am moving these bits to a separate doc in explanations.